### PR TITLE
Add hero refactor design doc redirect

### PR DESCRIPTION
**Pull request by chunhtai**
## Presubmit checklist

- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [ ] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -315,6 +315,7 @@
       { "source": "/go/go-router-v6-breaking-changes", "destination": "https://docs.google.com/document/d/1CMJwd5Moq_scvHf-trRldy9RW5MU7_Y8IB1aTrny-6s/edit", "type": 301 },
       { "source": "/go/handling-synchronous-keyboard-events", "destination": "https://docs.google.com/document/d/1rWXSjkb2ZKv-cpg26lVK0aZi4cVeXJ8j7YmSJdq2TOM/edit", "type": 301 },
       { "source": "/go/hermetic-xcode-installation", "destination": "https://docs.google.com/document/d/1EcXm4Woq48GR_ky07mEJka6NfV1vFBN2OHDEeGfPk34/edit?resourcekey=0-0Gjtt1I66mGJ8AwiCTlFNw", "type": 301 },
+      { "source": "/go/hero-refactor", "destination": "https://docs.google.com/document/d/17EG-Sgp7kHVetX-odJ5jrgnpBWj2lx097sYtrSP5xno", "type": 301 },
```
Comment (parlough) [inline firebase.json:318]: This doc doesn't seem to be public yet. Can you update it?

**Review by parlough — CHANGES_REQUESTED** [review]

File: firebase.json (hunk 1 above)
Comment (johnpryan) [reply firebase.json:318]: It's not public for me either

Comment (chunhtai) [reply firebase.json:318]: This is weird, I have to toggle the permission back and forth several times. This should be public now, I just test with another devices with non corp login

Comment (johnpryan) [reply firebase.json:318]: Weird, I still can't access it.

Comment (chunhtai) [reply firebase.json]: I changed to a different file. Let me know if this is still not working

**Review by johnpryan — APPROVED** [review]

**Pull request merged**